addons/survey/static/tests/components/question_page_one2many_field_tests.js

```diff
@@ -13,11 +14,15 @@ QUnit.module("QuestionPageOneToManyField", (hooks) => {
         serverData = {
             models: {
                 partner: {
```

Comment:
Could we have "survey" and "questions" instead of "partner" and "lines_sections" ?

It's really strange to read currently.